hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.2.21
+
+Release 2019-08-30
+
+### Correctness
+- (hail#6945) Fix `expand_types` to preserve the order of the data in a table
```

Comment:
to clarify: "to preserve ordering by key". Also note `to_pandas` and `to_spark` are affected.

#6958 is not a correctness fix.

Reply:
Error is a kind of incorrect 😉